engines/bastion_katello/app/assets/javascripts/bastion_katello/environments/details/views/environment-details.html

```diff
@@ -13,16 +13,27 @@ <h4 translate>Basic Information</h4>
     <dt translate>Label</dt>
     <dd>{{ environment.label }}</dd>
 
-    <dt translate>Registry Name Pattern</dt>
-    <dd bst-edit-textarea="environment.registry_name_pattern"
-        on-save="save(environment)"
-        readonly="denied('edit_lifecycle_environments', environment)">
-    </dd>
-
     <dt translate>Description</dt>
     <dd bst-edit-textarea="environment.description"
         on-save="save(environment)"
         readonly="denied('edit_lifecycle_environments', environment) || environment.library">
     </dd>
   </dl>
+
+  <h4 translate>Container Image Registry</h4>
```

Comment:
I think it belongs on the details page. It would be an uncommon pattern to have config of a resource on different page than Details. We could bring up w/ UX to consider moving both this option and the registry name pattern to the container images tab.

The _Docker_ term could be considered as separate issue to fix; I agree it should be changed. (Separate PR/commit in case needs to be backported to earlier release.)